Add a shortcut key and ability to persist connection target

eraya-msft opened the pull request:
- Add a shortcut key for the attach dialog.
- Persist last successful connection target as a setting.

eraya-msft committed:
Add a shortcut key for attach dialog. Persist last successful connection target as a setting.

MarcinJuraszek commented on src/WacVsTools.Core/AttachToWacProcess/AttachToWacProcessDialogModel.cs:
```diff
@@ -20,13 +26,18 @@ public class AttachToWacProcessDialogModel : INotifyPropertyChanged
+
+            var settingsManager = new ShellSettingsManager(ServiceProvider.GlobalProvider);
+            settingsStore = settingsManager.GetWritableSettingsStore(SettingsScope.UserSettings);
+            settingsStore.CreateCollection(CollectionPath);
```
What is this going to do when it already exist?

MarcinJuraszek replied:
I'm assuming it will do nothing, but can you add a comment here to make it easier to understand?

eraya-msft replied:
Yes, it skips the already existing collections.

eraya-msft replied:
> I'm assuming it will do nothing, but can you add a comment here to make it easier to understand?

sure, will add a comment to clarify.

eraya-msft committed:
Address code review comment

Added a code comment for clarification of CreateCollection call and bumped up the vsix version.
